db/local/objects/files.ts

```
@@ -0,0 +1,70 @@
+      request.onsuccess = () => {
+        resolve(request.result);
+      };
+
+      request.onerror = () => {
+        reject(request.error);
+      };
+    } catch (error) {
+      console.error('Error counting files:', error);
+      reject(error);
+    }
+  });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)